### ISSUE TITLE
Props not passed from PageContainer to specified layout

### ISSUE THREAD

**Issue by motleydev** [MoOx/phenomic#778]
PageContainer includes a wealth of routing props such as params, history, etc.

I'd like to be able to grab some of them and pass it down to the layout that is actually rendering such as Homepage or Post.

I think this has to be handled by passing props to the createElement call on line 232 of component.js in the PageContainer lib module.

```
!!page.error && PageError && _react2.default.createElement(PageError, page),
        !page.error && !page.loading &&
        Layout && _react2.default.createElement(
           Layout,
          _extends({
             ref: this.saveContentRef },
             page,
             props)) // <= Adding in props here 
```

**Comment by motleydev** [comment]
I was debating if this was a good idea or not since it would make the props on the page rather messy, but I think in this case it makes sense because I am defining that page as a singular rendered endpoint on the site, therefore I would potentially want to have all the routing props available to use.

**Comment by MoOx** [comment]
I guess it's not a bad idea, we can probably do that easily. Will take a look after the huge incoming release ;)

**Comment by bloodyowl** [comment]
Going to be fixed with #925
